src/profile-logic/profile-data.js

```diff
@@ -3371,6 +3375,136 @@ export function findAddressProofForFile(
   return null;
 }
 
+/**
+ * Calculate a lower bound for the function size, in bytes, of a native symbol.
+ * This is used when the symbol server does not return a size for a function.
+ * We need to know the size when we want to show assembly code for the function,
+ * in order to know how many bytes to disassemble.
+ * We estimate the size by finding the highest known address for this symbol in
+ * the frame table, and adding one byte (because the instruction at that address
+ * is at least one byte long).
```

Comment:
This is probably a very naive quesiton:
you write "at least one byte long", so this means that this can be longer, right?
In that case, shouldn't we take the largest possible size for an instruction instead?

Reply:
It could be longer, but this function computes a lower bound for the function size, so we should take the smallest possible size. Otherwise we might disassemble instructions from the next function by accident. We don't want to overrun.